java-checks/src/main/resources/org/sonar/l10n/java/rules/java/S6806.json

```diff
@@ -0,0 +1,23 @@
+{
+  "title": "Model attribute naming convention for Spring Expression Language (SpEL)",
+  "type": "BUG",
+  "status": "ready",
+  "remediation": {
+    "func": "Constant\/Issue",
+    "constantCost": "5min"
+  },
+  "tags": [],
+  "defaultSeverity": "Major",
+  "ruleSpecification": "RSPEC-6806",
+  "sqKey": "S6806",
+  "scope": "Main",
+  "quickfix": "unknown",
+  "code": {
+    "impacts": {
+      "MAINTAINABILITY": "MEDIUM",
+      "RELIABILITY": "MEDIUM",
+      "SECURITY": "LOW"
```

Comment:
I mentioned this on the RSPEC PR but I am not sure if this rule should add to the security impact since this is a self-inflicted pain and does not seem to be influenceable by user input